### DISCUSSION TITLE
Feature Request: Simplifying Line Breaks in Bio Section and Improving Text Readability Across Platforms

### DISCUSSION THREAD

**Discussion by haleygelato** [community/community#102932] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Now, if you want to add a line break in the Bio section of the web version, you have to manually insert the Unicode character 'ㅤ,' which is quite inconvenient. The ability to add line breaks is fundamental so I hope this feature can be added. Additionally, even if you add these Unicode spaces on the web, the text in the iOS app still appears crowded together. Hope there can be an improvement in readability. Thank you!

**Comment by Paulie-Aditya** [comment]
+1
